pkg/features/features.go

```diff
@@ -11,7 +11,7 @@ import (
 
 const (
 	// DisableGardenerServiceAccountCreation controls whether the gcp provider will create a default service account for VMs managed by MCM.
-	// beta: v1.29.0
+	// beta: v1.29.13
```

Comment:
You could revert this change. 

The version here is related to the provider-gcp version. It denotes that the corresponding feature gate is beta since provider-gcp@v1.29.0.